# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/elinks/hotfix-gpm.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- elinks-0.20.0/meson.build.vanilla	2026-08-05 23:56:34.184084011 +0300
+++ elinks-0.20.0/meson.build	2026-08-05 23:56:42.433772707 +0300
@@ -623,8 +623,8 @@
 endif
 
 if conf_data.get('CONFIG_GPM')
-    libgpmdeps = compiler.find_library('gpm', static: st)
-    deps += libgpmdeps
+#    libgpmdeps = compiler.find_library('gpm', static: st)
+#    deps += libgpmdeps
 endif
 
 if conf_data.get('CONFIG_SCRIPTING_PYTHON')

